yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -449,3 +449,17 @@ var percentTypeSyntax = issue.Issue{
 func NewPercentTypeSyntaxIssue(objectType string, objectName string, sqlStatement string) QueryIssue {
 	return newQueryIssue(percentTypeSyntax, objectType, objectName, sqlStatement, map[string]interface{}{})
 }
+
+var loDatatypeIssue = issue.Issue{
+	Type:       LARGE_OBJECT_DATATYPE,
+	TypeName:   "Unsupported datatype - lo",
+	Suggestion: "Large objects are not yet supported in YugabyteDB, no workaround available currently",
+	GH:         "https://github.com/yugabyte/yugabyte-db/issues/25318",
+	DocsLink:   "", //TODO
+}
+
+func NewLODatatypeIssue(objectType string, objectName string, SqlStatement string, colName string) QueryIssue {
+	issue := loDatatypeIssue
+	issue.TypeName = fmt.Sprintf("%s on column - %s", issue.TypeName, colName)
```

Comment:
I think we may have already discussed this, but I'm a little confused as to why the TypeName has to include the column. Shouldn't that go into objectName, perhaps?

Reply:
We could do something with Details here as well, but given that we currently have this format for other unsupported datatype cases, I think we go with this for now and modify it later.